src/Symfony/Bundle/SecurityBundle/Tests/Security/FirewallContextTest.php

```diff
@@ -36,7 +31,25 @@ public function testGetters()
 
         $context = new FirewallContext($listeners, $exceptionListener, $config);
 
-        $this->assertEquals(array($listeners, $exceptionListener), $context->getContext());
+        $this->assertEquals(array($listeners, $exceptionListener), $context->getListeners());
         $this->assertEquals($config, $context->getConfig());
     }
+
+    /**
+     * @expectedDeprecation Method Symfony\Bundle\SecurityBundle\Security\FirewallContext::getContext() is deprecated since version 3.3 and will be removed in 4.0. Use Symfony\Bundle\SecurityBundle\Security\FirewallContext::getListeners() instead.
+     * @group legacy
+     */
+    public function testGetContextTriggersDeprecation()
+    {
+        (new FirewallContext(array(), $this->getExceptionListenerMock(), new FirewallConfig('main', 'request_matcher', 'user_checker')))
+            ->getContext();
```

Comment:
you should also assert the return value, to ensure that the BC layer works fine (returning null would make your test pass, but the class would be broken)